### PR TITLE
Implement checkAccountHolder ressource for Account service

### PR DESCRIPTION
**Pull request by hammat**
**Description**
Make [Account checkAccountHolder](https://docs.adyen.com/api-explorer/#/Account/v6/post/checkAccountHolder) endpoint accessible from this library

### CODE REVIEW

File: src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    protected $endpoint;
+
+    /**
+     *
+     * @param Service $service
+     */
+    public function __construct($service)
+    {
+        $this->endpoint = $service->getClient()->getConfig()->get('endpointAccount') .
+            '/' . $service->getClient()->getApiAccountVersion() . 'checkAccountHolder';
```
Comment (acampos1916) [inline src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php:21]: Is this missing a forward slash before `checkAccountHolder`?

**Review by acampos1916 — COMMENTED** [review]
Thanks for this contribution @hammat

File: src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php (hunk 1 above)
Comment (hammat) [reply src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php]: You're right, my bad, just fixed it

**Review by AlexandrosMor — DISMISSED** [review]

File: src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Adyen\Service\ResourceModel\Account;
+
+use Adyen\Service;
+
+class CheckAccountHolder  extends \Adyen\Service\AbstractResource
```
Comment (peterojo) [inline src/Adyen/Service/ResourceModel/Account/CheckAccountHolder.php:7]:
```suggestion
class CheckAccountHolder extends \Adyen\Service\AbstractResource
```

**Review by acampos1916 — APPROVED** [review]

**Review by Morerice — APPROVED** [review]

**Comment by hammat** [comment]
Thank you for reviewing and merging.

Any chance to include this update within a 10.1.1 tag ?

**Pull request merged**